File: app/src/main/java/com/moviepass/ScreeningPosterClickListener.java
package com.moviepass;

import android.widget.ImageView;

import com.moviepass.model.Screening;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by anubis on 6/10/17.
 */

public interface ScreeningPosterClickListener {

    void onScreeningPosterClick(int pos, Screening screening, List<String> startTimes, ImageView shareImageView);

}

File: app/src/main/kotlin/com/moviepass/listeners/ScreeningPosterClickListener.kt
package com.moviepass.listeners

import android.widget.ImageView

import com.moviepass.model.Screening

/**
 * Created by anubis on 6/10/17.
 */

interface ScreeningPosterClickListener {

    fun onScreeningPosterClick(pos: Int, screening: Screening, startTimes: List<String>, shareImageView: ImageView)

}
